### PR TITLE
Get translations, galleries, and targetconfig from CDN

### PR DESCRIPTION
**Pull request by mmoskal**

### CODE REVIEW

File: pxtlib/emitter/cloud.ts [hunk 1]
```diff
@@ -43,17 +73,7 @@ namespace pxt.Cloud {
             options.headers["x-td-access-token"] = accessToken
```
Comment (pelikhan) [inline pxtlib/emitter/cloud.ts]: is this token dead?

**Comment by abchatra** [comment]
Isn’t browser caching works by ? being differentiator? Will now change in date ignored by browser cache as you use $?

File: pxtlib/emitter/cloud.ts (hunk 1 above)
Comment (mmoskal) [reply pxtlib/emitter/cloud.ts]: not sure; I guess we might want a token if we ever have a login

**Comment by mmoskal** [comment]
Browser cache will be invalidated every day, but the Etag should still work. $ vs ? won't make a difference, but before were were not appending date. I don't think browser would cache these things for more than a day anyways, as they have not cache expiration headers.

**Comment by pelikhan** [comment]
We cache translations in IndexedDb anyway.

**Comment by abchatra** [comment]
Looks good to me. Good to merge?

**Comment by mmoskal** [comment]
can only merge after cloud deployment

**Comment by mmoskal** [comment]
Houston, we have a problem: 
```
Access to XMLHttpRequest at 'https://pxtstage.azureedge.net/api/md/arcade/tutorials$targetVersion=0.14.46&cdn=20191204' from origin 'https://arcade.staging.pxt.io' has been blocked by CORS policy: Request header field if-none-match is not allowed by Access-Control-Allow-Headers in preflight response.
```

**Pull request merged**